src/sonic-build-hooks/scripts/post_run_buildinfo

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 
+IMAGENAME=$1
+
 . /usr/local/share/buildinfo/scripts/buildinfo_base.sh
 
 # Collect the version files
 collect_version_files $POST_VERSION_PATH
 
-[ -d $BUILD_VERSION_PATH ] && [ ! -z "$(ls -A $BUILD_VERSION_PATH)" ] && cp -rf $BUILD_VERSION_PATH/* $POST_VERSION_PATH
-rm -rf $BUILD_VERSION_PATH/*
+#Save the cache file for exporting it to host.
+tar -C ${PKG_CACHE_PATH} --exclude=cache.tgz -zcvf /cache.tgz .
```

Comment:
@Kalimuthu-Velappan 

BTW, what do you think about using pigz here?
In my tests I get a big improvements in time when use it instead of gzip:
https://github.com/sonic-net/sonic-buildimage/pull/12825

Reply:
Thanks for your suggestion. 
As you said, I have already used this in the binary optimization PR.